print('Hello git world! in test')

print a # error

for i in range(0,5):
    print('number',i+1)